apps/src/lab2/views/components/ResizeBar.tsx

```diff
@@ -53,21 +59,31 @@ const ResizeBar: React.FunctionComponent<ResizeBarProps> = ({
     ? moduleStyles.verticalBar
     : moduleStyles.horizontalBar;
 
+  const {onPointerDown, ...mainSeparatorProps} = separatorProps;
+
   return (
     <div className={classNames(moduleStyles.resizeBar, layoutClass)}>
+      {/* The visible bar */}
       <div
-        className={classNames(moduleStyles.grabbableDiv, grabbableClass)}
-        {...separatorProps}
+        className={classNames(moduleStyles.absoluteBar, resizingBarClass)}
```

Comment:
Could add a comment above line 65 that explains this is also the visible 1px bar vs. the 3px visible bar on hover.